lib/install.js

```diff
@@ -350,40 +341,39 @@ function install (gyp, argv, callback) {
         })
       }
 
+      //TODO: download appropriate node.lib
       function downloadNodeLib (done) {
-        log.verbose('on Windows; need to download `node.lib`...')
+        log.verbose('on Windows; need to download `' + release.name + '.lib`...')
         var dir32 = path.resolve(devDir, 'ia32')
           , dir64 = path.resolve(devDir, 'x64')
-          , nodeLibPath32 = path.resolve(dir32, 'node.lib')
-          , nodeLibPath64 = path.resolve(dir64, 'node.lib')
-          , nodeLibUrl32 = distUrl + '/v' + version + '/node.lib'
-          , nodeLibUrl64 = distUrl + '/v' + version + '/x64/node.lib'
+          , libPath32 = path.resolve(dir32, release.name + '.lib')
```

Comment:
How is this change going to interact with legacy deployments originally built using an older version of `node-gyp`? Will the old `node.lib` just be left there untouched until `node_modules` gets nuked?